variants/frontend-base/template.rb

```diff
@@ -98,4 +98,16 @@
   mini-css-extract-plugin
   sass
   sass-loader
+  @hotwired/turbo-rails
 ]
+
+# Setup Turbo
+yarn_add_dependencies %w[
+  @hotwired/turbo-rails
+]
```

Comment:
issue: this is installing the same package twice

```suggestion
```

Reply:
Doh. Typo from me. I won't keep your suggestion because I wanted Turbo to be in it's own part of the file but I will remove the duplication

Reply:
ok but that means we're running `yarn install` yet again which has it re-do a bunch more work - I know its not the biggest deal in the context of the template, but imo its still silly when the line immediately before it is doing an install...

I'll not block over it though 🤷